docs/docs-content/cluster-templates/modify-cluster-templates.md

```diff
@@ -0,0 +1,301 @@
+
+- An existing [cluster template](./cluster-templates.md).
+
+## Modify Cluster Templates
+
+Use the appropriate cluster template tab depending on the required modification.
+
+| **Tab**                                     | **Available Actions**                                                                                                                                                                                                  |
+| ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [**Overview**](#overview-tab)               | - Update your cluster to the latest version, regardless of the upgrade schedule specified in the maintenance policy.                                                                                                   |
+| [**Policies**](#policies-tab)               | - Add, update, replace, and remove referenced cluster templates and policies. <br /> - Modify the cluster profile version of a linked cluster profile.                                                                 |
+| [**Variable values**](#variable-values-tab) | - Get a high-level overview of all cluster profile variables deployed using the cluster profiles linked to the cluster template. <br /> - Perform batch updates for cluster variable values across specified clusters. |
+
+### Overview Tab
+
+Use the **Overview** tab to manually update clusters linked to your cluster template and to view the following
+information about your cluster template:
+
+- Basic metadata
+
+- Number of linked policies
+
+- Number of clusters linked to the cluster template
+
+- List of linked cluster profiles and their packs
+
+The option to manually trigger cluster updates is available only if a cluster template is linked to a cluster _and_
+there is a discrepancy between the cluster profile variable versions used in the cluster and the versions referenced in
+the template. This is caused when a cluster profile has been recently added to or removed from the cluster template, or
+the version of a cluster profile linked to a cluster template has been updated, but the upgrade window specified in the
+attached maintenance policy has not passed.
+
+To force cluster updates and bypass the scheduled maintenance window, select **Options > Upgrade now**.
+
+![Manually updating clusters attached to a cluster template](/cluster-templates_modify-cluster-templates_upgrade-now.webp)
+
+:::warning
+
+If there are any pending variable assignments, the banner **[N] cluster(s) pending variable assignment** is displayed.
+Even if you attempt to force an upgrade, the cluster will _not_ be updated when you select **Upgrade now**. Navigate to
+the [**Variable values**](#variable-values-tab) tab and assign a **New Value** to each variable before attempting the
+upgrade again.
```

Comment:
"Even if you attempt to force an upgrade, the cluster will _not_ be updated" --> it may be obvious but we must be clear in which clusters will not get upgraded. Here, the usage of "the cluster" is confusing.
Only the clusters with no pending variables assignement will not get upgraded

Reply:
```suggestion
If there are any pending variable assignments, the banner **[N] cluster(s) pending variable assignment** is displayed.
Even if you attempt to force an upgrade, any clusters without pending variable assignments will _not_ be updated when you select **Upgrade now**. Navigate to
the [**Variable values**](#variable-values-tab) tab and assign a **New Value** to each variable before attempting the
upgrade again.
```